nixos/modules/services/web-servers/nginx/default.nix

```
@@ -655,6 +655,16 @@ in
           Supported Nginx forks include `angie`, `openresty` and `tengine`.
           For HTTP/3 support use `nginxQuic` or `angieQuic`.
         '';
+        relatedPackages = [
+          "angie"
+          "angieQuic"
+          "nginxStable"
+          "nginxMainline"
+          "nginxQuic"
+          "nginxShibboleth"
```

```suggestion
          "nginxStable"
          "nginxMainline"
```
I would omit the packages that are just overrides of others with extra modules

---

This list matches `nixos/tests/nginx-variants.nix` and I was hoping for it to be able to reference this list from `options`, but I could not figure out how to do so

---

I also assume structured tools like the clan configuration editor would like to have all options enumerated

---

@SuperSandro2000 do you still block this?